iot-lab/iot-lab-yocto
# Linux open node

* remove obsolete recipes: udev and flash scripts
* rename open-a8-packagegroup to linux-node-packagegroup
* update all package groups list
* add rpi3 autotest image makefile target
* ensure old compat with <flash|reset|debug>_a8_m3 scripts with gateway-code generic programmer
* modify flash idle initrd script with gateway-code generic programmer (eg. open Linux nodes image)
* add shared symlink to A8 root home directory
* add gateway-code-server package with update-rc.d script & <start|stop>_dc_on scripts & postinstall dialout group
     * IoT-LAB gateway IoT-LAB or RPI3: install gateway-code & gateway-code-server package
     * IoT-LAB open A8 node or RPI3: install gateway-code package (eg. udev rules + generic programmer)
